src/object.di

```diff
@@ -632,17 +632,17 @@ private void _destructRecurse(E, size_t n)(ref E[n] arr)
 
 private string _genFieldPostblit(S)()
 {
+    import core.internal.convert : itoa;
     import core.internal.traits : hasElaborateCopyConstructor;
-    import core.internal.traits : FieldNameTuple;
 
     string code;
-    foreach(fieldName; FieldNameTuple!S)
+    foreach(i, FieldType; typeof(S.init.tupleof))
     {
```

Comment:
I tried that initially, but the static foreach introduces a new scope for `scope(failure)`, so it doesn't behave correctly.